graylog2-web-interface/src/stores/lookup-tables/LookupTablesStore.js

```diff
@@ -103,6 +112,34 @@ const LookupTablesStore = Reflux.createStore({
     return promise;
   },
 
+  getErrors(tableNames, cacheNames, dataAdapterNames) {
+    const request = {};
+    if (tableNames) {
+      request.tables = tableNames;
+    }
+    if (cacheNames) {
+      request.caches = cacheNames;
+    }
+    if (dataAdapterNames) {
+      request.data_adapters = dataAdapterNames;
+    }
+
+    const promise = fetch('POST', this._url('errorstates'), request);
+
+    promise.then((response) => {
+      this.trigger({
```

Comment:
Shouldn't this be persisted in the store's state? If it doesn't we can remove the variable from the store scope and return the initial state object in `getInitialState()`.